Makefile

```diff
@@ -7,7 +7,11 @@ export GOFLAGS
 GOPROXY ?=
 export GOPROXY
 
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.25
+
```

Comment:
I saw Daniel is doing similar work but moving things to 1.26.
Do we want to do it here already or do we prefer to do so in a followup PR?

Reply:
Testutils is now based on 1.26. E2E suite will require machine-api-operator to be updated first.